pybikes/publibike.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2010-2022, eskerda <eskerda@gmail.com>
+# Copyright (C) 2022, eUgEntOptIc44 (https://github.com/eUgEntOptIc44)
+# Distributed under the AGPL license, see LICENSE.txt
+
+import re
+import json
+
+from .base import BikeShareSystem, BikeShareStation
+from pybikes.utils import PyBikesScraper, filter_bounds
+from pybikes.contrib import TSTCache
+
+__all__ = ['Publibike', 'PublibikeStation']
+
+BASE_URL = 'https://{hostname}/v1/public/partner/stations'
+
+# caches the feed for 60s
+cache = TSTCache(delta=60)
+
+
+class Publibike(BikeShareSystem):
+    sync = True
+    unifeed = True # all 'networks' (instances) share the same feed
+
+    meta = {
+        'system': 'PubliBike',
+        'company': ['PubliBike AG'],
+        'source': 'https://api.publibike.ch/v1/static/api.html'
+    }
+
+    def __init__(self, tag, meta, city_uid, hostname='api.publibike.ch',
+                 bbox=None):
+        super(Publibike, self).__init__(tag, meta)
+        self.url = BASE_URL.format(hostname=hostname)
+        self.uid = city_uid
+        self.bbox = bbox
```

Comment:
Unused bbox argument

Reply:
@eskerda was leftover from the nextbike code that I used as a base. Now removed.